<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QDialog" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0.636, x2:0, y2:0.999591, stop:0 rgba(0, 0, 127, 255), stop:0.99435 rgba(0, 0, 0, 255));}</string>
   </property>
   <widget class="QLabel" name="label_login">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>661</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 900 36pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>CHOOSE WHO YOU ARE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_employee">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);
font: 900 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>EMPLOYEE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_customer">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);
font: 900 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>CUSTOMER</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
